fdbserver/BlobManifest.actor.cpp

```diff
@@ -163,11 +163,10 @@ class BlobManifest {
 			const BlobManifestFile& firstFile = *iter;
 			result.push_back(firstFile);
 			// search all following files belonging to same manifest
-			for (auto it = iter + 1; it != allFiles.end(); ++it) {
```

Comment:
fix for an correctness failure when allFiles has only 1 file and it never goes into this loop.